pkg/util/winutil/pdhutil/pdh_amd64.go

```diff
@@ -0,0 +1,25 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-2019 Datadog, Inc.
+// +build windows
+
+package pdhutil
+
+// Union specialization for double values
+type PDH_FMT_COUNTERVALUE_DOUBLE struct {
+	CStatus     uint32
+	DoubleValue float64
+}
+
+// Union specialization for 64 bit integer values
+type PDH_FMT_COUNTERVALUE_LARGE struct {
+	CStatus    uint32
+	LargeValue int64
+}
+
+// Union specialization for long values
+type PDH_FMT_COUNTERVALUE_LONG struct {
+	CStatus   uint32
+	LongValue int32
+}
```

Comment:
You might be able to use the same structs for both 32 bits and 64 bits as the struct size is the same.

Reply:
I tried it out, on 64 bit Go seems to pad out each member, so putting padding in between CStatus and LongValue ends up writing the value of LongValue into the padding variable, so we have to keep separate structure defs.  Good idea though